CHANGELOG.md

```diff
@@ -2,6 +2,12 @@ Note: This is in reverse chronological order, so newer entries are added to the
 
 Swift 5.6
 -----------
+
+* [#3890]
+
+  The fingerprint of a package is now being recorded when the package is first downloaded from a Git repository or package registry. Subsequent downloads must have fingerpints matching previous recorded values, otherwise it would result in build warnings or failures depending on settings. 
```

Comment:
```suggestion
To increase the security of packages, SwiftPM performs trust on first use (TOFU) validation. The fingerprint of a package is now being recorded when the package is first downloaded from a Git repository or package registry. Subsequent downloads must have fingerpints matching previous recorded values, otherwise it would result in build warnings or failures depending on settings. 
```